components/Issuer/DiplomaCredentialForm.vue

```diff
@@ -61,7 +61,7 @@
 <script setup lang="ts">
 import { ref } from 'vue';
 import type {DiplomaSchema} from "~/composables/VerifiableCredential";
-const emit = defineEmits(['DiplomaObjectCreated']);
+const emit = defineEmits(['DiplomaObjectCreated', 'AddToLog']);
```

Comment:
Why is it `AddToLog` here and `addToLog` in other places?

Reply:
That's a bug. I fixed it by removing this emit as it's not used in this component